hack/otel/compose.yaml

```
@@ -0,0 +1,27 @@
+name: moby-otel
+
+services:
+
+  jaeger:
+    image: jaegertracing/all-in-one:latest
+    ports:
+      - 16686:16686
+
+  aspire-dashboard:
+    image: mcr.microsoft.com/dotnet/nightly/aspire-dashboard
+    ports:
+      - 0.0.0.0:18888:18888
+    environment:
+      DOTNET_DASHBOARD_UNSECURED_ALLOW_ANONYMOUS: 'true'
+  
+  otelcol:
+    image: otel/opentelemetry-collector:latest
```

The `otel/opentelemetry-collector` image only contains a minimal set of processors, connectors, and no filters. So it's not particularly useful beyond just forwarding to multiple exporters.

Could you switch to `otel/opentelemetry-collector-contrib` instead? This would allow us to easily experiment with processing pipelines directly from this repo.

```suggestion
    image: otel/opentelemetry-collector-contrib:latest
```